.typos.toml

```diff
@@ -0,0 +1,9 @@
+[default.extend-identifiers]
+aks = "aks"
+AKS = "AKS"
+
+[default.extend-words]
+"ba" = "ba"
+
+[default]
+locale = "en-us"
```

Comment:
Are we using US English for this?

Reply:
I thought so, but I'll let @JonoCX make the call.

Reply:
Just as we are using NHS and Mind I think it makes sense to use GB English.

Reply:
British English 👍🏻